cluster/charts/rook-ceph/templates/deployment.yaml

```diff
@@ -94,11 +94,31 @@ spec:
         - name: ROOK_DISABLE_DEVICE_HOTPLUG
           value: "{{ .Values.disableDeviceHotplug }}"
         - name: ROOK_CSI_ENABLE_RBD
-          value: "{{ .Values.enableCsiRbdDriver }}"
+          value: "{{ .Values.csi.enableRbdDriver }}"
         - name: ROOK_CSI_ENABLE_CEPHFS
-          value: "{{ .Values.enableCsiCephfsDriver }}"
+          value: "{{ .Values.csi.enableCephfsDriver }}"
         - name: ROOK_CSI_ENABLE_GRPC_METRICS
-          value: "{{ .Values.enableCsiGrpcMetrics }}"
+          value: "{{ .Values.csi.enableGrpcMetrics }}"
+{{- if .Values.csi.cephcsi }}
```

Comment:
Either you need to check for `image`, or else you might set a blank image if they uncomment the parent value, but leave the `image` commented out.

Reply:
updated code to check for the image